example/cumulus-tf/cnm_response_task.tf

```
@@ -50,3 +50,9 @@ resource "aws_lambda_function" "cnm_response_task" {
 
   tags = local.tags
 }
+
+resource "aws_cloudwatch_log_group" "cnm_response_task" {
+  name              = "/aws/lambda/${aws_lambda_function.cnm_response_task.function_name}"
+  retention_in_days = lookup(var.cloudwatch_log_retention_periods, "cnmResponseTask_log_retention", var.default_log_retention_days)
```

Suggest to match the function_name (log group name matches the function_name).  `_log_retention` part isn't necessary.

https://github.com/nasa/cumulus/pull/3315/files#diff-27693bcf9b0bc94683ba213f789a8ed14c97e20a78594324c5d10b16dc1b4dc8R24

```suggestion
  retention_in_days = lookup(var.cloudwatch_log_retention_periods, "CnmResponse", var.default_log_retention_days)
```

---

1. we only need to worry about cumulus lambda/tasks.
2. No need to list the lambdas on the document, just indicate that the variable for the retention period should be lambda or task name, which is cloud watch log group name after the prefix, so we don't need to maintain the list.
3. example/cumulus-tf/variables.tf cloudwatch_log_retention_periods add a couple of lambdas/tasks from each module (such as tea-map-cache), default_log_retention_days set to lower value for our testing stacks.
I suggest you add all the lambda/tasks to cloudwatch_log_retention_periods for your testing locally.
4. default_log_retention_days and cloudwatch_log_retention_periods are optional parameters, so place them in the # optional section in variables.tf and terraform.tfvars.example

---

everything under tf-modules/* are cumulus lambdas and tasks